caddyhttp/push/handler.go

```diff
@@ -58,26 +58,41 @@ outer:
 }
 
 func (h Middleware) servePreloadLinks(pusher http.Pusher, headers http.Header, links []string) {
+outer:
 	for _, link := range links {
-		parts := strings.Split(link, ";")
+		resources := strings.Split(link, ",")
 
-		if link == "" || strings.HasSuffix(link, "nopush") {
-			continue
-		}
+		for _, resource := range resources {
+			parts := strings.Split(resource, ";")
 
-		target := strings.TrimSuffix(strings.TrimPrefix(parts[0], "<"), ">")
+			if resource == "" || strings.HasSuffix(resource, "nopush") {
+				continue
+			}
 
-		err := pusher.Push(target, &http.PushOptions{
-			Method: http.MethodGet,
-			Header: headers,
-		})
+			target := strings.TrimSuffix(strings.TrimPrefix(parts[0], "<"), ">")
+
+			if h.IsRemoteResource(target) {
+				continue
+			}
 
-		if err != nil {
-			break
+			err := pusher.Push(target, &http.PushOptions{
+				Method: http.MethodGet,
+				Header: headers,
+			})
+
+			if err != nil {
+				break outer
+			}
 		}
 	}
 }
 
+func (h Middleware) IsRemoteResource(resource string) bool {
```

Comment:
Does this have to be exported? If it's not used outside the package, let's un-export it. Add a godoc comment either way.